DatabaseMocker/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Backend;
+using Models;
+
+await using var context = new AtriaContext();
+Console.WriteLine(string.Join(", ", context.Users.Select(x => x.Email)));
+
+WebserviceEntry entry = new WebserviceEntry {
+    Name = "Test",
+    ShortDescription = "SD",
+    FullDescription = "FUlld",
+    Link = new Uri("https://google.de"),
```

Comment:
inconsistent with changes in WebserviceEntry 
```suggestion
    Link = "https://google.de",
```